opencga-app/app/conf/hadoop/hadoop-ssh.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+
+if [ -z ${HADOOP_SSH_USER} ] ; then
+  echo "Undefined HADOOP_SSH_USER" 1>&2
+  exit 1
+fi
+
+if [ -z ${HADOOP_SSH_HOST} ] ; then
+  echo "Undefined HADOOP_SSH_HOST" 1>&2
+  exit 1
+fi
+
+SSHPASS_CMD=
+if [ -z ${SSHPASS} ] ; then
+  # If empty, assume ssh-key exists in the system
+  SSHPASS_CMD=""
+else
+  # If non zero, use sshpass command
+  SSHPASS_CMD="sshpass -e"
+fi
+
+SSH_OPTS="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
```

Comment:
Adding `ssh -o ServerAliveInterval=60` should be sufficient to keep the connection open. We'll have to test to see whether we need to tweak the HDI ssh config in the init container or not.

Reply:
Good point. Let's see if it's enough